docs/CONFIG.md

```diff
@@ -5,29 +5,63 @@ While most of EVE's configuration is done via receiving a configuration object f
+In such cases it is required to deliver the initial, also known as *bootstrap*, device configuration off-line. The recommended method (over some legacy mechanisms described below) is to prepare a single-use EVE installation medium, carrying the bootstrap config for the target device inside the EVE partition.
 
-## Controlling EVE behavior at boot via legacy configuration management
+The bootstrap configuration is modeled using the same Protobuf message that models device configuration delivered on-line: [EdgeDevConfig](../api/proto/config/devconfig.proto). Just like in the on-line config delivery, an instance of `EdgeDevConfig` is protobuf-encoded and put into the `AuthContainer` envelope alongside a signature (see [OBJECT-SIGNING.md](../api/OBJECT-SIGNING.md)). For the off-line signature verification (that must pass for the device to accept the configuration), it is necessary to further wrap `AuthContainer` together with the signing and all intermediary controller certificates inside [BootstrapConfig](../api/proto/config/devconfig.proto). Finally, `BootstrapConfig` should be protobuf-encoded and written in this binary format into the EVE partition of a single-use EVE installer as `bootstrap-config.pb`. During the EVE installation, this file is copied into the CONFIG partition.
+
+The format of the bootstrap configuration is rather complex and intentionally binary to disincentivise users from preparing and editing the configuration manually. Instead, it is expected that the process of preparing and exporting the bootstrap configuration is done by the controller and the tools that it provides. To bake an exported bootstrap config into a single-use EVE installer, use [eve-gen-single-use-installer.sh](../tools/eve-gen-single-use-installer.sh) script (run with no arguments to print the script usage).
+
+Currently, it is expected that the bootstrap configuration will only contain those attributes of `EdgeDevConfig` which are in one way or another related to device connectivity:
+
+* `networks`
+* `configItems`
+* `systemAdapterList`
+* `deviceIoList`
+* `vlans`
+* `bonds`
+* `config_timestamp`
+
+Anything else will be silently ignored.
+
+The above allows to configure one or multiple management interfaces, as well as non-management interfaces, and can specify static IP and DNS configuration (for environments where DHCP is not used), plus the LTE configuration. In addition, it can specify proxies using several different mechanisms (explicitly, using WPAD, etc.). [Object level encryption](./OBJECT-LEVEL-ENCRYPTION.md) is not yet supported inside the bootstrap config, meaning that WiFi settings with the network password have to be left out it. Currently, it is therefore not possible to onboard device with WiFi-only connectivity using this mechanism.
```

Comment:
"not yet" implies that WiFi will be added soon, but that isn't doable if we want the WiFi credentials to be encrypted; can't do that until the controller knows the device's ECDH certificate so it can encrypt for that target.
So I would strike "not yet" from the sentence.

Reply:
Done